### PR TITLE
add GRADLE_USER_HOME env, fixes cache and wrapper volumes mounts

### LINKED ISSUE #778: Downloading gradlew each docker run of android-builder
Wrote a similar Dockerfile and now I see downloading of gradlew despite I have volumes for gradle and gradlew caches
image

Glad if you can check if this issue is reproducible in your infrastructure
image

### PR DESCRIPTION
**Pull request by dsvoronin**
Fixed #778

### CODE REVIEW

File: ci/_main.sh [hunk 1]
```diff
@@ -105,6 +105,7 @@ function runInBuilder() {
         --env TZ="Europe/Moscow" \
         --env LOCAL_USER_ID="$USER_ID" \
         --env BINTRAY_GPG_PASSPHRASE="$BINTRAY_GPG_PASSPHRASE" \
+        --env GRADLE_USER_HOME=/gradle \
```
Comment (eugene-krivobokov) [inline ci/_main.sh:108]: This env is optional. Should we consider it?

Comment (dsvoronin) [reply ci/_main.sh:108]: You mean we should fail on it's abscence in entrypoint?

Comment (eugene-krivobokov) [reply ci/_main.sh:108]: This is an option. 
The alternative is to repeat default behavior: use `$USER_HOME/.gradle` if not set.

Comment (dsvoronin) [reply ci/_main.sh:108]: There is a problem with mounting into folder of not yet created user; 
i didn't go deeper, please check `entrypoint.sh` changes

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**